pkg/identityserver/gormstore/end_device.go

```diff
@@ -56,6 +56,7 @@ type EndDevice struct {
 	PictureID *string `gorm:"type:UUID;index:end_device_picture_index"`
 
 	ActivatedAt *time.Time `gorm:"default:null"`
+	LastSeen    *time.Time `gorm:"default:null"`
```

Comment:
`LastSeenAt`

Reply:
Please extend the tests in `pkg/identityserver/storetest/end_device_store.go` to cover this field.